mitosheet/mitosheet/step_performers/graph_steps/graph_delete.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+from copy import copy
+from typing import Any, Dict, List, Optional, Set, Tuple
+
+from mitosheet.state import State
+from mitosheet.step_performers.step_performer import StepPerformer
+from mitosheet.types import GraphID
+
+
+class GraphDeleteStepPerformer(StepPerformer):
+    """
+    Deletes a graph from everywhere in the step.
+    """
+
+    @classmethod
+    def step_version(cls) -> int:
+        return 1
+
+    @classmethod
+    def step_type(cls) -> str:
+        return 'graph_delete'
+
+    @classmethod
+    def step_display_name(cls) -> str:
+        return 'Deleted a Graph'
+    
+    @classmethod
+    def step_event_type(cls) -> str:
+        return 'graph_delete_edit'
+
+    @classmethod
+    def saturate(cls, prev_state: State, params: Dict[str, Any]) -> Dict[str, Any]:
+        graph_id = params['graph_id']
+        old_graph_tab_name = prev_state.graph_data[graph_id]
+        params['old_graph_tab_name'] = old_graph_tab_name
+        return params
```

Comment:
Do we need this? I think this is used in the similar delete step for the data frame, but only because we need the name in the transpile step; since we don't have a transpile step here, this can go.

Reply:
I'm using the name in the describe step to make the comment in the steps taskpane informative.